integration_test/soak_test/launcher.go

```diff
@@ -0,0 +1,191 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build integration_test
+
+/*
+The launcher command launches a VM and begins a soak test on it.
+
+Specifically, it installs the Ops Agent and a Python program that
+logs to a specific file that the Ops Agent is watching.
+
+This command is configured by the following environment variables,
```

Comment:
Why does this use environment variables and not command-line flags?

Reply:
I dunno, a few reasons:

- env variables are easy
- certain env variables that we use are already supplied by kokoro, no need to explicitly copy them into flags in the .sh file
- the integration tests currently use env variables instead of flags
- env variables and flags have pretty much the same syntax anyway, not sure what the advantage of flags would be?

Reply:
It's just non-idiomatic for Go. If you're consuming Kokoro-defined variables then I guess you should do the locally normal thing.